test/errors/implicitSubModule.1.good

```diff
@@ -0,0 +1,9 @@
+─── warning in test/errors/implicitSubModule.chpl:4 [ImplicitFileModule] ───
+  An implicit module named 'implicitSubModule' is being introduced to contain this file-scope code.
+  The following is the first file-scope statement:
+      |
+    4 | use List;
+      |
+  The implicit module 'implicitSubModule' is being created because the above code is outside of any module declarations (e.g., 'module M').
+  Note that all of the file's contents will be placed into the new 'implicitSubModule' module.
```

Comment:
This is minor—in the "I'm not confident it's worth fixing sense"—but it did occur to me and give me pause, so I'll mention it:  My initial read of this warning's description made me think that if the initial structure of my code had been:

```chapel
module TheModule {
  var y = 1;
}
var x = 1;
```

that this warning is telling me it would become:

```chapel
module implicitSubModule {
  var y = 1;
  var x = 1;
}
```

rather than:

```chapel
module implicitSubModule {
  module TheModule {
    var y = 1;
  }
  var x = 1;
}
```

Because I'm a Chapel expert, I could correct that interpretation, knowing what it meant.  But if I were a new user, I might not...

If you agreed, a simple fix might be to say that "Note that all the file's contents, including its explicitly defined modules, will be placed into the new ..."

[nevermind this:  I think I was scrolling the wrong direction in the file diff?]  ~The first time I read this, I assumed that "e.g., `module M`" was referring to the real module name in the source code—but it's not, right(?), it's just using 'M' as a completely generic module name.  Would it be easy/possible to put the/an actual module name from the source code in here instead "(e.g., `module TheModule`?)".~

We could even combine these two ideas and adjust the modest proposal just above, saying "the file's contents—including module 'TheModule'—into the new 'implicitSubModule' module."  Or "the file's contents into the new 'implicitSubModule' module, including 'TheModule' as a sub-module".  Or something like that.

Again, none of these seem like show-stoppers to me.  But the question of what happens to existing module structure when placing it into the implicit module does occur to me, particularly with my "naive user" hat on.